pyne/pyne_version.py

```diff
@@ -0,0 +1,26 @@
+import os
+
+# Set PYNE version
+PYNE_MAJOR_VERSION = 0
+PYNE_MINOR_VERSION = 7
+PYNE_PATCH_VERSION = 6
+
+PYNE_VERSION = f'{PYNE_MAJOR_VERSION}.{PYNE_MINOR_VERSION}.{PYNE_PATCH_VERSION}'
```

Comment:
One final thought here - PyNE requires Python >= 3.5.2, but f-strings aren't officially supported until 3.6. We should probably use `.format` here.

Reply:
Good call (or should we up the min version of python??)

Reply:
Probably should. No time to determine what the right min version should be at the moment though.

Reply:
OpenMC recently moved to a minimum python version of 3.6 perhaps the same motivating reasons apply
https://github.com/openmc-dev/openmc/pull/1860